File: app/src/main/java/com/saessac/SplashActivity.java
package com.saessac;

import android.os.Bundle;
import androidx.activity.ComponentActivity;
import androidx.core.splashscreen.SplashScreen;


public class SplashActivity extends ComponentActivity {
    @Override
    protected void onCreate(Bundle savedInstanceState) {
        SplashScreen.installSplashScreen(this);
        super.onCreate(savedInstanceState);
        SaessacUI.openActivity(this, LoginRegisterActivity.class, R.anim.slide_in_right, R.anim.slide_out_left);
    }
}

File: app/src/main/java/com/example/saessac/SplashActivity.kt
package com.saessac

import android.os.Bundle
import android.content.Intent
import androidx.activity.ComponentActivity
import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
import com.example.saessac.SaessacUI

class SplashActivity : ComponentActivity()
{
    override fun onCreate(savedInstanceState: Bundle?)
    {
        installSplashScreen()
        super.onCreate(savedInstanceState)
        SaessacUI.OpenActivity(this, LoginRegisterActivity::class.java)
    }
}